r-publishing.Rmd

```diff
@@ -116,7 +119,7 @@ Try visiting: `https://{{your_username}}.github.io/sharing-work/report.html`.  Y
 
 You could send people the link, `https://{{your_username}}.github.io/sharing-work/report.html`, but it is often nicer to send them the shorter version without the file name: `https://{{your_username}}.github.io/sharing-work`.  You can try this now, but it won't work --- you'll see a message in your browser like: "404- File not found".  This shorter URL points to a directory as opposed to a file.  By default, when a server receives a request for a directory, it looks for a file to display with a default name --- usually `index.html`.  In your case there is no file called `index.html` so there is nothing to display.
 
-If you would like the contents of `report.html` to be displayed as the homepage of your project, then rename `report.Rmd` to `index.Rmd`.  You'll then need to regenerate the HTML file, remove `index.html`, commit, and push your changes.   For work that is communicated easily in one page, this would be a good option. 
```

Comment:
Correct me if I am misunderstanding, but I think this is just a typo since that is the file they just generated, right?

Reply:
You are right.  If anything was to be "removed" perhaps I meant `report.html`?